FreeRTOS/FreeRTOS-Kernel
# Xtensa support updated and moved to Partner-Supported submodules

Remove legacy Xtensa port; leave README with pointer to latest partner-maintained code

Description
-----------
The Xtensa FreeRTOS port has been consolidated in the Partner-Supported submodules, along with numerous updates and bug-fixes.
The old location now contains a README indicating where to find the latest version of the Xtensa port and test code.

Test Steps
-----------
- Tested via various demo and stress tests as provided through the Partner-Supported-Demos submodule.
- Regression testing on many different LX7, LX8, and NX configurations.

Checklist:
----------


- [ x ] I have tested my changes. No regression in existing tests.
- [ x ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
